libri-whitepaper.tex

```diff
@@ -54,13 +54,15 @@ \section{Introduction}
 % 	- problems it does not try to solve (right now)
 % - other medical-data related projects
 
-The US healthcare system has become very decentralized, with no single provider of healthcare and thus no single place where everyone's health data lives. The insurance side of the industry is also decentralized, with many different public and private payers, large and small. This decentralization creates many challenges for patients. Libri is an effort to solve one: patients have almost no visibility or control over their own health data.
+Inexorably, large organizations across all domains are becoming more adapt at creating, collecting, and processing data relevant to their operations in search of valuable insights. Increasingly, these enterprises are looking to pool their data to enhance its analytical value, solve collective action problems, and provide more comprehensive understandings of system-wide events. These nascent data consortiums are emerging among financial institutions (for instance, in support of anti-money laundering activities), educational institutions, and also within the healthcare sector.
 
-Many people have experience this problem first hand when they need to switch doctors because of a move or change in insurance or when they need to coordinate among a number of different healthcare providers, as in an orthopedic proceedure like a knee replacement. Unfortunately, this process still involves numerous phone calls, faxes, and often hard-copy paper or CDs. Despite the existance of efficient, secure forms of data transfer for over 20 years, these arcane, analog data transfer methods still comprise the majority of how health data is managed today. 
+It is notable, that participation in these collaborative data schemes affords benefit to the enterprise operators (albeit not directly monetary) in the form of simplified access to additional valuable data (at lower operational cost but with increased analytical potential), or by the ability to add enhanced value-add services to their customers via the comprehensive data reach (e.g., data portals).
```

Comment:
I like the message. Here's my attempt to tighten up the wording a bit: `Participation in these coordinated data repositories benefits all related organizations through simplified, standardized access to data for improved operations and richer analytics. The comprehensive data reach of these collaborations also benefit consumers, who can now track and control their data movement across sectors fragmented by many private and public institutions.`